Bigtable/src/Instance.php

```diff
@@ -273,20 +273,40 @@ public function create(array $clusterMetadataList, array $options = [])
             // `$clustersArray` must be keyed by the cluster ID.
             $clustersArray[$clusterId] = $value;
         }
+        $info = ['displayName' => $displayName ];
+        if ($type != null) {
```

Comment:
Use the `!==` operator for more precise comparisons.

That said, I liked the old way of doing this better. I imagine you made this change so that you weren't sending potentially empty fields over the wire.

You could use `ArrayTrait::arrayFilterRemoveNull()` to clear out empty values:

```php
        $operation = $this->connection->createInstance([
            'parent' => $projectName,
            'instanceId' => $this->id,
            'instance' => $this->arrayFilterRemoveNull([
                'displayName' => $displayName,
                'type' => $type,
                'labels' => $labels
            ]),
            'instance' => $info,
            'clusters' => $clustersArray
        ] + $options);
```

I'm ok with creating the `$info` var separately, but I prefer constructing the filtered array to the multiple conditionals. :)

Reply:
@jdpedrie Thanks for the information about this, I will update using arrayFilterRemoveNull.

Reply:
@jdpedrie arrayFilterRemoveNull passing empty array, is there any function to filter empty array? For now I added empty array ( labels) as part of expected call argument in Instance unit test, but ideally I would like to avoid forwarding empty array if its not passed.